### PR TITLE
Tweak project-level copy a bit

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,9 +14,7 @@ Please read https://reactjs.org/ and the Code of Conduct before opening an issue
 <a name="bug"/>
 ## Think You Found a Bug?
 
-Please provide a test case of some sort. Best is a pull request with a failing test. Next is a link to codepen/jsbin or repository that illustrates the bug. Finally, some copy/pastable code is acceptable.
-
-If you don't provide a test case, the issue will be closed.
```
Comment (taion) [inline CONTRIBUTING.md]: Maybe? I think it's okay to ask for test cases when they're helpful, but sometimes it's simple enough to not need a test case.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Thank you for contributing!
 
 Have a usage question?
 ======================
-Questions *might* be closed without comment, sometimes we just don't have time for them. Before you ask one, here are some resources to get help first:
+The issue tracker isn't the best place for usage questions. This format is not well-suited for Q&A, and questions here don't have as much visibility as they do elsewhere. Before you ask a question, here are some resources to get help first:
```
Comment (taion) [inline ISSUE_TEMPLATE.md]: I feel like this is a bit more gentle – "you should go to other places because they're better".

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# react-router [![Travis][build-badge]][build] [![npm package][npm-badge]][npm]
+# React Router [![Travis][build-badge]][build] [![npm package][npm-badge]][npm]
 
-[`react-router`](https://www.npmjs.com/package/react-router) is a complete routing library for [React](https://facebook.github.io/react).
+React Router is a complete routing library for [React](https://facebook.github.io/react).
```
Comment (taion) [inline README.md]: Just a bunch of copy-editing tweaks; I updated repo description to say "A complete routing library for React" as well.

File: README.md [hunk 4]
```diff
@@ -10,19 +10,18 @@ React Router keeps your UI in sync with the URL. It has a simple API with powerf
 ### Docs & Help
 
 - [Tutorial – do this first!](https://github.com/reactjs/react-router-tutorial)
-- [Guides and API docs](https://github.com/reactjs/react-router/tree/master/docs)
+- [Guides and API docs](/docs)
```
Comment (taion) [inline README.md]: These links don't need to be fully qualified. npm resolves these correctly as well, e.g. for the `/SPONSORS.md` link below.

File: CONTRIBUTING.md (hunk 1 above)
Comment (timdorr) [inline CONTRIBUTING.md]: The pessimist in me says people are going to ignore this no matter what you write.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (timdorr) [inline ISSUE_TEMPLATE.md]: Positive reinforcement is more effective than negative.

File: README.md (hunk 3 above)
Comment (timdorr) [inline README.md]: What was it before?

File: README.md (hunk 4 above)
Comment (timdorr) [inline README.md]: Depending on how we do the branching for 3.0, we might need to make this a fully-qualified URL again to link to the right branch for docs.

File: README.md (hunk 3 above)
Comment (taion) [inline README.md]: "for React.js"

**Pull request merged**